### PR TITLE
Support setting copyright on collection

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -231,7 +231,7 @@ def edit_copyright
 
   # save the copyright form
   def update
-    change_set = ItemChangeSet.new(@cocina)
+    change_set = @cocina.is_a?(Cocina::Models::Collection) ? CollectionChangeSet.new(@cocina) : ItemChangeSet.new(@cocina)
```
Comment (ndushay) [inline app/controllers/items_controller.rb:234]: random thought, not for this PR:  should we have `cocina-obj.collection?` `.dro?` and `.apo?` convenience methods provided in cocina-models?

File: spec/requests/item_update_spec.rb [hunk 2]
```diff
@@ -18,36 +18,71 @@
...
+        expect(object_client).to have_received(:update)
+          .with(params: updated_model)
+        expect(Argo::Indexer).to have_received(:reindex_pid_remotely)
+        expect(response.code).to eq('303')
+      end
     end
 
-    it 'sets the new collection' do
-      patch "/items/#{pid}", params: { item: { copyright: 'in public domain' } }
+    context 'with an collection' do
```
Comment (ndushay) [inline spec/requests/item_update_spec.rb:55]: `a` not `an`

**Review by ndushay — APPROVED** [review]
looks good.  One quibble you can take or ignore and a random question.

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (jcoyne) [reply app/controllers/items_controller.rb:234]: I believe they already exist.

Comment (jcoyne) [reply app/controllers/items_controller.rb:234]: https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/models/checkable.rb

**Pull request merged**